### PR TITLE
first edit

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-9a14f307c7/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a2e89ac7fa/how-to/)

**Comment by WilliamLoy** [comment]
@cyrichardson ready for you :)

**Comment by cyrichardson** [comment]
@WilliamLoy On it.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-86fd609b33/how-to/). :bow:

File: content/exchange/manually-configure-mac-mail-for-email-hosted-on-exchange-2013.md [hunk 1]
```diff
@@ -1,53 +1,35 @@
...
+last_modified_date: '2019-03-28'
+last_modified_by: William Loy
 product: Microsoft Exchange
 product_url: exchange
 ---
 
-This article describes how to set up your Microsoft Exchange 2013
-email account with Mac Mail.
+This article describes how to set up your Microsoft&reg; Exchange&reg; 2013
+email account with Mac&reg; Mail.
```
Comment (cyrichardson) [inline content/exchange/manually-configure-mac-mail-for-email-hosted-on-exchange-2013.md]: Did not find Mail to be a registered trademark here: https://www.apple.com/legal/intellectual-property/trademark/appletmlist.html

**Comment by cyrichardson** [comment]
@WilliamLoy Editorial changes made. Please review.

**Comment by WilliamLoy** [comment]
LGTM!

**Pull request merged**